network/modules/udp/client_socket.py

```diff
@@ -0,0 +1,117 @@
+"""
+Wrapper for UDP client socket operations.
+"""
+
+import socket
+
+from .socket_wrapper import UdpSocket
+
+
+class UdpClientSocket(UdpSocket):
+    """
+    Wrapper for UDP client socket operations.
+    """
+
+    __create_key = object()
+
+    def __init__(
+        self,
+        class_private_create_key: object,
+        socket_instance: socket.socket,
+        server_address: tuple,
+    ) -> None:
+        """
+        Private Constructor, use create() method.
+        """
+
+        assert class_private_create_key is UdpClientSocket.__create_key
+        super().__init__(socket_instance=socket_instance)
+        self.server_address = server_address
```

Comment:
should this be private?

Reply:
To be fair, it's not really needed, so I'll just delete it.